templates/components/products/price.html

```diff
@@ -31,30 +32,31 @@
                 </div>
             {{/if}}
             {{#if price.without_tax}}
-                <abbr title="{{lang 'products.excluding_tax'}}">{{lang 'products.price_with_tax' tax_label=price.tax_label}}</abbr>
+                <abbr title="{{lang 'products.including_tax'}}">{{lang 'products.price_with_tax' tax_label=price.tax_label}}</abbr>
```

Comment:
🙃 im not sure whats going on here: the if check block is `{{#if price.without_tax}}` but then we're showing including tax?

Reply:
Yep, this is specifically for the use case of displaying both including and excluding tax prices on the page. When the condition passes we have both price including and excluding tax in our context. In such a case we want to display identifying text after the price `(including tax)` and `(excluding tax)`, respectively.

Using this specific line as an example, we have immediately before this point printed the price including tax. So _if_ the context also has a price excluding tax, then we need to print `(Including tax)` to the page to sit right after the including tax price we printed a few lines above.